README.md

```diff
@@ -18,15 +19,15 @@ This is meant to be a custom cog to be added to a [Red DiscordBot](https://githu
 ## Adding a New Joke
 + New jokes can usually be added as a new sub-class of the Joke class in the jokes directory.
 + The easiest way to sublcass the Joke object is to look at the existing jokes, but take care to
- 1. Change the `super().__init__("new_joke_name", joke_chance)` to the name of the joke and the default chance of occurance [0.0,100.0]
+ 1. Change the `super().__init__("new_joke_name", joke_chance)` to the name of the joke and the default chance of occurrence [0.0,100.0]
  2. Implement the joke within the `_make_joke` method.
  3. Edit `jokes/__init__.py` to import your joke and add an instance of it to the `JOKES` dictionary
   - The proper key for the new instance is "JokeNameJoke"
  4. Add a short description of this joke to this README.
 
 ### Why This Structure?
-The purpose is to have a standarized method of turning a user's message into a joke. 
+The purpose is to have a standardized method of turning a user's message into a joke. 
 Dad does this by randomizing the list of jokes, and then iterating through each joke until either he can make a joke with the message or he runs out of jokes.
 This can be seen in ``on_message`` method in `dad.py`.
 Further the `Joke` class adds the to the list of options that can be set for each guild.
-This allows guilds to modify the probablity that a particular joke will occur, without extra code needing to be written for that.
+This allows guilds to modify the probability that a particular joke will occur, without extra code needing to be written for that.
```

Comment:
I appreciate this spelling check. X 3

Reply:
You've got to love the VS code spell check plug in I have. actually it sucks